src/main/java/org/opensearch/security/configuration/DlsFlsFilterLeafReader.java

```diff
@@ -395,16 +397,6 @@ private void finishVisitor(StoredFieldVisitor visitor) {
         }
     }
 
-    @Override
```

Comment:
I believe this is already covered in the override for `storedFields()` which returns a [DlsFlsStoredFields](https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/configuration/DlsFlsFilterLeafReader.java#L355-L371) that overrides this method.